plugins/pa11y/readme.md

```diff
@@ -0,0 +1,45 @@
+# Description 
+ Pa11y is your automated accessibility testing pal. It runs accessibility tests on your pages via the command line or Node.js, so you can automate your testing process. Pa11y runs automatically on the `test:review` and `test:staging` hooks, using the information about the app from the state.
+
+# Pa11y documentation
+ `https://github.com/pa11y/pa11y`
+
+ # Installation and usage
+    With Tool Kit [already set up](https://github.com/financial-times/dotcom-tool-kit#installing-and-using-tool-kit), install this plugin as a dev dependency:
+
+    npm install --save-dev @dotcom-tool-kit/pa11y
+    And add it to your repo's .toolkitrc.yml:
+
+    plugins:
+    - '@dotcom-tool-kit/pa11y'
+
```

Comment:
**Issue:** We are missing markdown formatting that will make this section easier to read. Please see the markdown code for the comment below to see the missing formatting.

Reply:
With Tool Kit [already set up](https://github.com/financial-times/dotcom-tool-kit#installing-and-using-tool-kit), install this plugin as a dev dependency:

```
    npm install --save-dev @dotcom-tool-kit/pa11y
```

And add it to your repo's `.toolkitrc.yml`:

```
plugins:
    - '@dotcom-tool-kit/pa11y'
```